tasks/main.yml

```diff
@@ -8,3 +9,36 @@
   tags:
     - vars
 
+- name: Install the VDO packages
+  ansible.builtin.dnf:
+    name: "{{ vdo_pkgs_list }}"
+    state: installed
+
+- name: Create the volume group
+  community.general.lvg:
+    vg: "{{ vdo_vg_name }}"
+    pvs: "{{ vdo_pv_disks }}"
+
+- name: Setup VDO storage pool
+  community.general.lvol:
+    vg: "{{ vdo_vg_name }}"
+    lv: "{{ vdo_lv_name }}"
+    size: "{{ vdo_phys_size }}"
```

Comment:
Can we have also a subsequent lvol run to set the lv size accordingly ? so if we want to increase underlying size we can also reflect a new virtualsize on the vdo lv itself

Reply:
Yep, I'm just adding that now. Will do the FS type too.